content/shmem_pe_accessible.tex

```diff
@@ -23,11 +23,11 @@
 \apidescription{
     \FUNC{shmem\_pe\_accessible} is  a  query routine  that indicates  whether  a
     specified \ac{PE} is accessible via \openshmem from the local \ac{PE}. The
-    \FUNC{shmem\_pe\_accessible} routine returns \CONST{TRUE} only if  the  remote
+    \FUNC{shmem\_pe\_accessible} routine returns a value indicating whether the remote
     \ac{PE} is a process  running from the same executable  file as the local
-    \ac{PE}, indicating that full \openshmem support for symmetric data objects
-    (that reside in the static memory and symmetric heap) is available, otherwise it
-    returns \CONST{FALSE}.  This routine may be particularly useful for hybrid
+    \ac{PE}, thereby indicating whether full support for symmetric data objects,
+    which may reside in either static memory or the symmetric heap, is available.
+    This routine may be particularly useful for hybrid
```

Comment:
@jdinan I think this note about hybrid programming is better suited for the "Notes" subsection, but it also ties into whether the Altix references should be removed (see https://github.com/openshmem-org/specification/issues/161).

Reply:
Thanks, I'll capture a note for this in https://github.com/openshmem-org/specification/issues/161.